.travis.yml

```diff
@@ -30,26 +30,22 @@ before_install:
   - pip install -U pip wheel setuptools
   - pip install -U $DJANGO
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - ./runflake8
-  - ./runcheckmigration
-  - ./runisort
   - npm install -g jslint
   - jslint django_loci/static/django-loci/js/*.js
 
 install:
   - pip install -e .
+  - python setup.py -q develop
```

Comment:
this is not necessary (`pip install -e .` is the equivalent using pip), why did you add it? I only spotted it now

Reply:
@nemesisdesign Otherwise `django_loci` module isn't set-up, so makemigrations check fails

Reply:
@waleko `pip install -e .` is equivalent to `python setup.py -q develop`, so we're doing twice the same operation, if without this additional line it fails it's likely because some other reason which we would be covering this way

Reply:
@nemesisdesign My bad... I took a one more look at it and it turned out, that I've put `openwisp-utils-qa-checks` before `pip install -e .`